assets/src/domain/eventEditor/ui/ticketAssignmentsManager/data/useOnSubmitAssignments.ts

```diff
@@ -42,16 +57,30 @@ const useOnSubmitAssignments = () => {
 			if (ticketsToUpdate.length < datesToUpdate.length) {
 				ticketsToUpdate.forEach(([id, possibleRelation]) => {
 					const datetimes = possibleRelation?.datetimes || [];
-					updateTicket({ id, datetimes });
+					const input: UpdateTicketInput = { id, datetimes };
+					// if an entry exists in changed quantity map
+					// lets use this oppurtunity to update the quantity here
+					// to reduce the number of mutation requests
+					if (ticketsWithChangedQuantity?.[id]) {
+						input.quantity = ticketsWithChangedQuantity?.[id];
+						// mark the ticket as already updated
+						updatedTickets.push(id);
+					}
+					updateTicket(input);
 				});
 			} else {
 				datesToUpdate.forEach(([id, possibleRelation]) => {
 					const tickets = possibleRelation?.tickets || [];
 					updateDatetime({ id, tickets });
 				});
 			}
+
+			// now we finally update the ticket quantities
+			Object.entries(ticketsWithChangedQuantity).forEach(([id, quantity]) => {
+				updateTicket({ id, quantity });
+			});
```

Comment:
do we need to check `updatedTickets` first ?

Reply:
Fixed